docs/.storybook/manager-head.html

```diff
@@ -1,3 +1,26 @@
+<script>document.title = "UI Kit";</script>;
 <script>
-  document.title = "UI Kit"; window.STORYBOOK_GA_ID = "UA-38285631-31";
+  function injectScript(url, attributes = {}, onLoad) {
+    var tag = document.createElement('script');
+    tag.type = 'text/javascript';
+    tag.src = url;
+    Object.keys(attributes).forEach((key) => {
+      tag.setAttribute(key, attributes[key]);
+    });
+
+    if (onLoad) tag.onload = onLoad;
+      document.head.appendChild(tag);
+    };
+
+    // Inject the cookie consent scripts only if the page is served on a `*.commercetools.com` domain.
+  if (window && window.location.host.includes('.commercetools.com')) {
```

Comment:
Maybe this is an obvious question but I'll ask anyway 😅
We are injecting two scripts here right? 
One is the consent while the other is the template correct?

Reply:
I basically just copied the code that @emmenko linked in the slack-conversation leading to this request. As far as I managed to figure out, the first script attempts to makes sure that cookies are blocked automatically and the second one is the actual app that displays the UI and allows consenting.